src/proofs/SmtLibWriter.cpp

```diff
@@ -91,5 +92,8 @@ void SmtLibWriter::writeInstanceToFile( IFile &file, const List<String> &instanc
 
 String SmtLibWriter::signedValue( double val )
 {
-    return val > 0 ? std::to_string( val ) : "( - " + std::to_string( abs( val ) ) + " )";
+    unsigned precision = ( unsigned ) std::log10 ( 1 / GlobalConfiguration::DEFAULT_EPSILON_FOR_COMPARISONS );
+    std::stringstream s;
+    s << std::fixed << std::setprecision( precision ) << abs( val );
+    return val >= 0 ? String ( s.str() ).trimZerosFromRight() : String( "( - " + s.str() ).trimZerosFromRight() + " )";
```

Comment:
No space between a function name and the ensuing (
String ( s.str() --> String( s.str()
Likewise with std::log10 ( 1 ...